src/helper/userHelpers.js

```diff
@@ -0,0 +1,37 @@
+const reqlib = require('app-root-path').require;
+
+const { NotFound } = reqlib('src/errors');
+
+const getUser = (app, userId) =>
+	app
+		.service('users')
+		.get(userId, {
+			query: {
+				$populate: 'roles',
+			},
+		})
+		.then((user) => {
+			if (user === null) {
+				throw new Error('User not found.');
+			}
+			return user;
+		})
+		.catch((err) => {
+			throw new NotFound('Can not fetch user.', err);
+		});
+
+const testIfRoleNameExist = (user, ...roleNames) => {
+	if ((user.roles[0] || {}).name === undefined) {
+		throw new Error('Role is not populated.');
+	}
+	return user.roles.some(({ name }) => roleNames.includes(name));
+};
```

Comment:
why do you need this general function if you only ever call it with the constant `'superhero'` as second argument?

Reply:
Also, the role will not change (probably ever), so you can grab the role id on service start and look for that id in the user roles without populate

Reply:
Ok, my intention was to take existing code, extract to helper file and use it.